bindings/python/sample.py

```diff
@@ -22,11 +22,11 @@ def test_ks(arch, mode, code, syntax=0):
 
 # test symbol resolver
 def test_sym_resolver():
-    def sym_resolver(symbol, value):
+    def sym_resolver(symbol, p_value):
         # is this the missing symbol we want to handle?
         if symbol == "_l1":
             # put value of this symbol in @value
-            value = 0x1002
+            p_value.contents.value = 0x1002
```

Comment:
oh now i see that this interface does not look good. would be simpler for user to return the value, rather than set the content value like this. 

the reason is that this resolver must return `bool` value to indicate if it sets the symbol. perhaps we should change the API a bit: this function either returns symbol value, or `None` when it does not care. so the function above will be changed to be:

```
    def sym_resolver(symbol):
        # is this the missing symbol we want to handle?
        if symbol == "_l1":
            # we handled this symbol, so return true
            return 0x1002

        # we did not handle this symbol, so return None
        return None
```

what do you think?